server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -49,7 +48,6 @@ static TransportVersion def(int id) {
         return new TransportVersion(id);
     }
 
-    @UpdateForV9(owner = UpdateForV9.Owner.CORE_INFRA) // remove the transport versions with which v9 will not need to interact
```

Comment:
This one isn't done yet, we need to remove all the version constants up to the one for 8.18.0 (at least `8_840_00_1`, maybe higher) first.

Reply:
ok, could you leave a `// TODO` comment here in the code that links to `ES-10337`?